stravalib/model.py

```diff
@@ -459,8 +722,9 @@ class ActivityPhoto(BackwardCompatibilityMixin, DeprecatedSerializableMixin):
     created_at: Optional[datetime] = None
     created_at_local: Optional[datetime] = None
     location: Optional[LatLon] = None
-    urls: Optional[Dict] = None
-    sizes: Optional[Dict] = None
+    urls: Optional[dict[str, str]] = None
+    # TODO - is this a float return?
+    sizes: Optional[dict[str, float]] = None
```

Comment:
These two annotations are my best guess for the 3.8 test failing. Tried to replace them by `Dict` from the typing module. The use of `dict` for annotation has only been introduced in 3.9

Reply:
oh gosh - i forgot about that. you are right - it's running now locally. thank you!!